src/chrome/breakOnLoadHelper.ts

```diff
@@ -137,16 +90,16 @@ export class BreakOnLoadHelper {
 
         // Important: For the logic that verifies if a user breakpoint is set in the paused location, we need to resolve pending breakpoints, and commit them, before
         // using committedBreakpointsByUrl for our logic.
-        let anyPendingBreakpointsResolved = await this.resolvePendingBreakpointsOfPausedScript(pausedLocation.scriptId);
+        await this._chromeDebugAdapter.breakpointsResolvedDefer(pausedLocation.scriptId).promise;
 
         const pausedScriptUrl = this.getScriptUrlFromId(pausedLocation.scriptId);
         // Important: We need to get the committed breakpoints only after all the pending breakpoints for this file have been resolved. If not this logic won't work
-        const committedBps = this._chromeDebugAdapter.committedBreakpointsByUrl.get(pausedScriptUrl);
+        const committedBps = this._chromeDebugAdapter.committedBreakpointsByUrl.get(pausedScriptUrl) || [];
         const anyBreakpointsAtPausedLocation = committedBps.filter(bp =>
             bp.actualLocation.lineNumber === pausedLocation.lineNumber && bp.actualLocation.columnNumber === pausedLocation.columnNumber).length > 0;
 
-        // If there were any pending breakpoints resolved and any of them was at (1,1) we shouldn't continue
-        if (anyPendingBreakpointsResolved && anyBreakpointsAtPausedLocation) {
+        // If there were any breakpoints at (1,1) we shouldn't continue

```

Comment:
"at (1,1)" is this still accurate? It can be anywhere now?

Reply:
// If there were any breakpoints at this location (Which generally should be (1,1)) we shouldn't continue